packages/react-native-sdk/src/utils/push/utils.ts

```diff
@@ -3,6 +3,10 @@ import { FirebaseMessagingTypes } from './libs/firebaseMessaging';
 import { ExpoNotification } from './libs/expoNotifications';
 import { NonRingingPushEvent } from '../StreamVideoRN/types';
 import { PushNotificationiOSType } from './libs/iosPushNotification';
+import {
+  NotificationTrigger,
+  PushNotificationTrigger,
+} from 'expo-notifications';
```

Comment:
🛑 this should not be done, we should not be importing types for an optional dependency